### ISSUE TITLE
no-literal-string does not report string literals inside TypeScript namespaces

### ISSUE THREAD

**Issue by BiosSun** [edvardchen/eslint-plugin-i18next#161]
**Description**

String literals inside a TypeScript namespace block are never reported by the no-literal-string rule, regardless of configuration (mode: 'all', words, etc.).

**Reproduction**

```ts
export namespace Something {
    export function foo() {
        return '中文'  // no warning
    }
}
```

ESLint config:

```ts
'i18next/no-literal-string': [
    'warn', 
    { 
        mode: 'all', 
        words: { include: ['.*[\\u4e00-\\u9fa5].*'] } 
    }
]
```

**Expected**

A warning is reported on the string literal.

**Actual**

No warning is reported.
